controllers/storagecluster/blackbox_exporter.go

```diff
@@ -0,0 +1,547 @@
+/*
```

Comment:
Please add unit tests to verify creating/deleting the resources.

Reply:
can we have it in followup PR? we already have a lot in this?

Reply:
Unit tests really should be planned with all PRs, but if separate must be done soon, otherwise it will be forgotten.